.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Please do not ask general questions here! There are [comunnity
+contact](https://github.com/qtile/qtile#community) options for that.
+
+# Issue description
+
+A brief discussion of what failed and how it failed. A description of what you
+tried is helpful, i.e. "When I use lazy.kill() on a window nothing happens"
+instead of "Closing windows doesn't work".
+
+# Qtile version
+
+Please include the exact commit hash of the version of Qtile that failed.
```

Comment:
People installing through their distribution package manager won't know the commit hash for sure

Reply:
Then they need to find it out :). This seems like useful information for debugging. Now, one thing I do notice is that qtile --version doesn't print it, so we should fix that.

Reply:
Yes, let's fix that first and say to past the output here